pkg/scheduler/framework/plugins/noderesources/most_allocated.go

```diff
@@ -25,7 +25,7 @@ import (
 // based on the maximum of the average of the fraction of requested to capacity.
 //
 // Details:
-// (cpu(MaxNodeScore * sum(requested) / capacity) + memory(MaxNodeScore * sum(requested) / capacity)) / weightSum
+// ((cpu(MaxNodeScore * sum(requested) * cpuWeight / capacity) + memory(MaxNodeScore * sum(requested) * memoryWeight / capacity)) + ...) / weightSum
```

Comment:
nits:
I guess `...` at the last means the resources other than CPU and memory, right?
If so, I prefer to remove the parentheses on `( cpu() + memory() )` because other resources should be treated equally with cpu and memory. 

```suggestion
// (cpu(MaxNodeScore * sum(requested) * cpuWeight / capacity) + memory(MaxNodeScore * sum(requested) * memoryWeight / capacity) + ...) / weightSum
```